### ISSUE TITLE
Procedure entry point CRYPTO_atomic_read could not be located in the dynamic link library

### ISSUE THREAD

**Issue by franzd001** [OpenShot/openshot-qt#4467]
A:  I have checked... issue does not currently exist

**Describe the bug:**

A:
![2021-10-05 17_57_17-openshot-qt exe - Entry Point Not Found](https://user-images.githubusercontent.com/89428616/136114851-19d35ac7-7463-4d77-9325-2b9aee9b86b9.png)

**Steps to reproduce the behavior:**
A:  This error occurs after double-clicking the App Icon (starting the application).  The application does not open/appear on screen until clicking on the OK button of the Error window.

**Expected behavior:**

A:  No errors on application startup, etc.

**System Details:**
- OpenShot Version [e.g. 2.4.3]: {{ log_context.openshot_qt_version }}
- Operating System / Distro: [e.g. Windows 10, Linux Mint 17.1]

A:  OpenShot 2.6.1 - x64, Windows 10 64-bit - 20H2 (19042.1237)

**Log Files:**
If you are experiencing a crash, please collect and attach logs of the problem.  Log files can be found in the `.openshot_qt` folder in your user home directory. Log files over 2 MB in size will need to be compressed, please attach a ZIP file instead of the raw logs.

**Exception / Stacktrace:**
If you are experiencing a crash or strange output on the command line, please attach a small snippet of the exception here, but please keep it as small as possible (for readability).

**Screenshots:** (Optional)
If applicable, add screenshots to help explain your problem. To include screenshots, copy/paste them here, drag the image(s) into this form, or click the link below this edit field to load images saved to your computer. All images are public, so please don't post screenshots containing personal information.

**Comment by franzd001** [comment]
[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/7289806/openshot-qt.log
[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/7289807/libopenshot.log)
)

**Comment by ferdnyc** [comment]
### TL;DR

You don't need to read the step-by-step of my process (unless you want to), the outcome was:
1. My last comment after I'd finished investigating [requested](https://github.com/OpenShot/openshot-qt/issues/4467#issuecomment-935621691),
     > can you please check whether 
     > ```text
     > C:\Program Files\OpenShot Video Editor\libssl-1_1-x64.dll
     > ```
     > exists on your computer?
2. A complete uninstall and reinstall of OpenShot might fix this, since it seems like there are files missing.

#### Original comment

Hmm. Clearly OpenShot is picking up the wrong `libssl-1_1-x64.dll`, I assume that Intel directory is in your `%PATH%`? Even still, OpenShot's own paths should be getting consulted prior to that one, I believe.

A workaround for the short term would be to remove the directory indicated from the `PATH` environment variable. (I promise you, you don't need it there, and it's pretty rude of software to insert itself like that on install even though many installers do exactly that.) You can edit the contents of `%PATH%` by:
1. Going to Settings > System > About
2. Choosing "Advanced system settings" from "Related settings" at the bottom of the page
3. Clicking "Environment Variables..." at the bottom of the window that comes up
4. Choosing "Path" in the list at the top half of the next window (that's the list of variables for your user account, only), and clicking the Edit... button. Another window will come up that looks like this: 
    ![image](https://user-images.githubusercontent.com/538020/136152941-1d7311e3-30d5-4f1d-a7cd-2fdbe63bde4b.png)

You can rearrange items on the list with "Move up/down" or remove entries with "Delete", removing that Intel directory should hopefully get OpenShot working again.

I'm going to look deeper into this and see if I can figure out why that'd be necessary in the first place, since like I said OpenShot **should** be finding its own libraries before ones in random path locations. But this is the quickest way to solve the problem on your machine in the short term.

**Comment by ferdnyc** [comment]
Actually, now that I look, you might not find that directory in the list I mentioned, because it might be only in the **SYSTEM** path (that's the list at the _bottom_ half of the window where I told you to click in the top half.) I'm saying that, because it's there in **MY** Windows 10 installation as well:

![image](https://user-images.githubusercontent.com/538020/136153551-2f65435f-731c-4af1-bc95-b4fde845c60f.png)

...But for whatever reason, on my system it's not interfering with OpenShot being able to launch.

**Comment by ferdnyc** [comment]
Huh. Interesting. My "Intel(R) Management Engine Components" doesn't _contain_ an "iCLS" subdirectory, so there's no libssl file there that _could_ interfere. Well, that's... inconclusive.

![image](https://user-images.githubusercontent.com/538020/136154648-beb54021-056a-4a17-9df3-78b6ce12a515.png)

**Comment by ferdnyc** [comment]
I've verified that `libssl-1_1-x64.dll` exists in `C:\Program Files\OpenShot Video Editor\`, though, and that OpenShot is picking it up from that location on my system.

It's possible your OpenShot install is _missing_ that DLL for some reason, so Windows is just loading whatever copy it can find on the PATH (as it's supposed to). You might want to try just completely uninstalling and reinstalling OpenShot to see if that fixes things. Perhaps somehow that file was deleted, or just never got installed the first time.

**Comment by ferdnyc** [comment]
If nothing else, though, can you please check whether `C:\Program Files\OpenShot Video Editor\libssl-1_1-x64.dll` exists on your computer? It **_should_** have been installed with OpenShot, and if it is present where it's supposed to be OpenShot _should_ be loading it from that location, not anywhere else on the system.

**Comment by qamanuel** [comment]
It was also missing from my folder, uninstalled and reinstalled. I just copied one from elsewhere after reading this.

Openshot 2.6.1 
Microsoft Windows 11 Home 10.0.22000 x64-based PC

**Comment by DChike** [comment]
I can confirm this Issue.

At start up a window open and close quickly and after that this error window pop up, after clicking OK OpenShot Video Editor opens and seems to work without problems.

If the folder "C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS" is renamed/removed, this error does not show.

**Comment by DChike** [comment]
First thing first: there are two copies of libssl-1_1-x64.dll in the program subfolder, why?

Second, for some reason the error occur at a second attempt to load libssl-1_1-x64.dll, after it was already loaded before. after it fails this instance of libssl-1_1-x64.dll is unloaded and the program continues as usual.

From Visual Studio debugger output (second load):
```
'openshot-qt.exe' (Win32): Loaded 'C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS\libssl-1_1-x64.dll'. 
'openshot-qt.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140.dll'. 
The thread 0x3504 has exited with code 0 (0x0).
The thread 0x5eb8 has exited with code 0 (0x0).
The thread 0x3f28 has exited with code 0 (0x0).
Exception thrown at 0x00007FF88CDD2346 (ntdll.dll) in openshot-qt.exe: 0xC0000139: Entry Point Not Found.
'openshot-qt.exe' (Win32): Unloaded 'C:\Windows\System32\vcruntime140.dll'
'openshot-qt.exe' (Win32): Unloaded 'C:\Program Files\Intel\Intel(R) Management Engine Components\iCLS\libssl-1_1-x64.dll'
```
I temporary copied the dll from OpenShot  lib to Intel's directory, attached is Visual Studio debug output
[debug.txt](https://github.com/OpenShot/openshot-qt/files/7567614/debug.txt)
